src/kernel/src/arch/amd64/interrupt.rs

```diff
@@ -1086,3 +1092,24 @@ pub fn set(state: bool) {
     flags.set(RFlags::FLAGS_IF, state);
     x86::bits64::rflags::set(flags);
 }
+
+pub fn allocate_interrupt_vector(
+    _pri: InterruptPriority,
+    _opts: InterruptAllocateOptions,
+) -> Option<DynamicInterrupt> {
+    // TODO: Actually track interrupts, and allocate based on priority and flags.
+    set_interrupt(
+        64,
```

Comment:
Yes, I'm going to whine about the magic number.  Please give it a name.

Reply:
I'm planning to, in another PR, implement an actual interrupt vector allocation mechanism. I didn't do it in this one because I wanted to get the changes to userland APIs out first.